bundle/bundle.go

```diff
@@ -127,8 +127,9 @@ type Manifest struct {
 
 // WasmResolver maps a wasm module to an entrypoint ref.
 type WasmResolver struct {
-	Entrypoint string `json:"entrypoint,omitempty"`
-	Module     string `json:"module,omitempty"`
+	Entrypoint string             `json:"entrypoint,omitempty"`
+	Module     string             `json:"module,omitempty"`
+	Metadata   []*ast.Annotations `json:"metadata,omitempty"`
```

Comment:
Should be encapsulate `[]*ast.Annotations` inside a struct ? That way we would have the flexibility to add more fields to the WASM metadata in the future.

Reply:
Alternatively, we see the `WasmResolver` as the container of metadata, and just add stuff to the top-level. The reason for calling it `Metadata` here, and not `Annotations` is for consistency with us talking about a collection of annotations as "metadata"; such as starting a block of annotations in Rego with the `METADATA` marker.

Reply:
In Rego, we create annotations inside `METADATA` blocks; and we can access that info in rules through the `rego.metadata.*` built-ins. But in the docs, we pretty much use `metadata` and `annotations` interchangeably. Also, since this is a list of annotations - where each `ast.Annotations` is a set of annotations, it might be less confusing to call it something else.
I see however that the already existing `metadata` attribute one level up in the `manifest` could cause some confusion; and I'm open to change this to `Annotations`/`annotations`. I'm 50/50 myself 😄 .